server/src/db/connection.ts

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+
+/* eslint-disable no-console */
+export const createConnection = (config) => {
```

Comment:
can we add type for config?

Reply:
I've added a type for config to shared types:
```Javascript
interface Conn {
    username: string;
    passwd: string;
    host: string;
    port: string;
    databaseName: string;
    connectionOptions: string;
}

interface MongoConnection {
    conn: Conn;
    uri: string;
    databaseName: string;
    reconnectInterval: number;
}

interface Config {
    appName: string;
    isProduction: boolean;
    mongoConnection: MongoConnection;
}
```